lib/content_block_tools/presenters/base_presenter.rb

```diff
@@ -29,12 +29,15 @@ def render
             content_block: "",
             document_type: content_block.document_type,
             content_id: content_block.content_id,
+            field_names: field_names.to_s,
```

Comment:
Might be better to `join` this with a comma, rather than presenting it as an array

Reply:
good shout, done!